lib/hooks/index.js

```diff
@@ -104,6 +106,8 @@ function findModuleVersion(request, parent, load) {
 
 function activate(agent) {
 
+  logger = agent.logger_;
```

Comment:
Decided it is easier to make it public than pass the logger in parallel everywhere the agent goes.